tests/pandas_postprocessing_tests.py

```diff
@@ -607,3 +617,103 @@ def test_prophet_incorrect_time_grain(self):
             periods=10,
             confidence_interval=0.8,
         )
+
+    def test_boxplot_tukey(self):
```

Comment:
![image](https://user-images.githubusercontent.com/5727938/97743075-60716380-1aa2-11eb-9019-f95808320a39.png)
there was a bug with 3rd quantile, will test it out once conflicts are resolved

Reply:
Good catch, let me take a look

Reply:
I meant prior to your diff, will test now if that is still an issue

Reply:
Actually it is still a problem
![image](https://user-images.githubusercontent.com/5727938/97772477-d27a9480-1b04-11eb-9818-5a19595d5ef6.png)
Data for the bucket:

[f] 1000-2499 |  7251270
[f] 1000-2499 |  1820590.6666666667
[f] 1000-2499 |  1187242.142857143
[f] 1000-2499 |  1151020
e.g. look like outlier was not excluded for quartile calculations

Reply:
Is this with Tukey whiskers? Interesting edge case with so few observations, I'll replicate with some other commonly used stats package to see what it would give for a similar case.